mmcv/cnn/bricks/padding.py

```diff
@@ -0,0 +1,32 @@
+import torch.nn as nn
+
+PADDING_CONFIG = {
+    'zero': nn.ZeroPad2d,
+    'reflect': nn.ReflectionPad2d,
+    'replicate': nn.ReplicationPad2d,
+}
+
+
+def build_padding_layer(cfg, *args, **kwargs):
+    """Build padding layer.
+
+    Args:
+        cfg (None or dict): The padding layer config, which should contain:
+            - type (str): Layer type.
```

Comment:
We may need to add contents about what we currently support in the docstrings thus the users know what can be used and what needs to be extended.